frontend/src/premium/eagle-eye/eagle-eye-routes.js

```diff
@@ -1,12 +1,13 @@
 import Layout from '@/modules/layout/components/layout.vue';
 import Permissions from '@/security/permissions';
 import { store } from '@/store';
+import config from '@/config';
 
 const EagleEyePage = () => import(
   '@/premium/eagle-eye/pages/eagle-eye-page-wrapper.vue'
 );
 
-export default [
```

Comment:
Maybe might be better to exclude eagle eye from here `frontend/src/modules/index.ts`
So that you can exclude the entire module, routes and store

Reply:
Good idea! I've updated the code